.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,8 @@
+-   id: tmt-test-lint
+    name: tmt tests lint
+    entry: bash -c "git ls-files --error-unmatch .fmf/version && tmt tests lint --source $@" PAD
```

Comment:
The `git ls-files --error-unmatch .fmf/version` works only relatively to the current directory, right? So this means you cannot `git commit` from a deeper directory without a hook error?

Reply:
You are right. Time for dedicated command? (which will report real path of .fmf/version from git root). 
AFAIK It has to be python script installed by setup.py if hook is python type.

Reply:
nice, this does the trick `entry: bash -c "git ls-files --error-unmatch $(python3 -c 'import tmt; print(tmt.Tree(\".\").root)')/.fmf/version && tmt tests lint --source $@" PAD`